<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Compress</class>
 <widget class="QDialog" name="Compress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source file name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination file name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression algorithm:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseSourceButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseDestinationButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QComboBox" name="algorithmButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>huffman</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lz4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>paq9a</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="sourceFileNameLine">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="destinationFileNameLine">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoAlgorithmCheckBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose automatically</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elapsed time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="elapsedTimeLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 sec</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ETA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="estimatedTimeLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0 sec</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Start compression</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>warning</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
